MainModule/Server/Commands/Players.lua

```diff
@@ -312,7 +312,7 @@ return function(Vargs, env)
 
 		ServerSpeed = {
 			Prefix = Settings.PlayerPrefix;
-			Commands = {"serverspeed", "serverping", "serverfps", "serverlag", "tps"};
+			Commands = {"serverspeed", "serverping", "serverfps", "serverlag", "tps", "fps"};
```

Comment:
This may conflict with a `!fps` command (for measuring client FPS) should one be added in the future. (Refer: #880)

Reply:
k